api/controllers/HomeController.js

```diff
@@ -8,7 +8,8 @@
 module.exports = _.mapValues({
   async index (req, res) {
     const boxes = await Box.find({owner: req.user.name});
-    res.view('home/view', {boxes});
+    const prefs = await UserPreferences.find({user: req.user.name});
```

Comment:
Let's keep it as it is just now, and once we have alpha up and running we can see if having all this is causing issues. I doubt it will, as the boxes is the only big one, and we need that to show anything anyway.